pod/podfile/views.py

```diff
@@ -594,38 +595,16 @@ def user_share_autocomplete(request):
 
 @login_required(redirect_field_name="referrer")
 def remove_shared_user(request):
-    if is_ajax(request):
-        foldid = request.GET.get("foldid", 0)
-        userid = request.GET.get("userid", 0)
-        if foldid == 0 or userid == 0:
-            return HttpResponseBadRequest()
-        folder = UserFolder.objects.get(id=foldid)
-        user = User.objects.get(id=userid)
-        if folder.owner == request.user or request.user.is_superuser:
-            folder.users.remove(user)
-            folder.save()
-            return HttpResponse(status=201)
-        else:
-            return HttpResponseBadRequest()
+    if request.is_ajax:
+        return update_shared_user(request, "remove")
     else:
         return HttpResponseBadRequest()
 
 
 @login_required(redirect_field_name="referrer")
 def add_shared_user(request):
-    if is_ajax(request):
-        foldid = request.GET.get("foldid", 0)
-        userid = request.GET.get("userid", 0)
-        if foldid == 0 or userid == 0:
-            return HttpResponseBadRequest()
-        folder = UserFolder.objects.get(id=foldid)
-        user = User.objects.get(id=userid)
-        if folder.owner == request.user or request.user.is_superuser:
-            folder.users.add(user)
-            folder.save()
-            return HttpResponse(status=201)
-        else:
-            return HttpResponseBadRequest()
+    if request.is_ajax:
```

Comment:
idem, c'est is_ajax ou request.is_ajax. Si tu l'import dans ton module python, autant l'utiliser ;) lequle faut-il utiliser ?

Reply:
HttpRequest.is_ajax() is deprecated starting with Django version 3.1.
Donc à mon avis mieux vaut utiliser is_ajax(request) qu'on maintient nous-même.

Reply:
oui, je corrige